Feature/230 prefix handling

abdessamad-abdoun opened the pull request.

abdessamad-abdoun committed:
#230  Implementation of a prefix handling abstract class for parser/serializer

abdessamad-abdoun committed:
#230  Implementation of a prefix handling abstract class for parser/serializer

remiceres commented on src/main/java/fr/inria/corese/core/next/impl/io/serialization/base/AbstractGraphSerializer.java:
```diff
@@ -20,6 +20,7 @@
+import fr.inria.corese.core.next.impl.common.prefix.PrefixHandler;
 import fr.inria.corese.core.next.impl.common.vocabulary.*;
 import fr.inria.corese.core.next.impl.exception.ParsingErrorException;
 import org.slf4j.Logger;
```
some imports a useless

remiceres reviewed (changes requested):
The changes look good to me, thanks for the work. However, this introduces several new SonarQube issues, especially regarding entrySet usage, clone(), and exception handling. It would be better to avoid introducing new warnings and fix the existing ones in the modified file so that the overall number decreases with each change.

Could you address these before we merge?

abdessamad-abdoun committed:
#230 revue
 Implementation of a prefix handling abstract class for parser/serializer

remiceres committed:
Add Eclipse plugin so VS Code detects generated sources

VS Code wasn’t indexing `build/generated-src`, causing false missing-class
errors. Adding the `eclipse` plugin fixes the classpath so VS Code loads the
generated sources correctly.
